src/routes/authRoute.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const authController = require('../controllers/authController.js');
+const { catchError } = require('../utils/catchError.js');
+
+const authRouter = new express.Router();
+
+authRouter.post('/registration', catchError(authController.register));
+
+authRouter.get(
+  '/activation/:activationToken',
+  catchError(authController.activate),
+);
+
+authRouter.post('/login', catchError(authController.login));
+authRouter.get('/refresh', catchError(authController.refresh));
```

Comment:
Consider changing the HTTP method for the `/refresh` endpoint from `GET` to `POST` to align with RESTful practices, as it modifies the server state by issuing new tokens.